bin_tests/tests/crashtracker_bin_test.rs

```diff
@@ -121,9 +121,13 @@ fn test_crash_tracking_bin(crash_tracking_receiver_profile: BuildProfile, mode:
     );
     assert_eq!(
         serde_json::json!({
-          "signum": 11,
-          "signame": "SIGSEGV",
+          "code": 1,
+          "codename": "SEGV_MAPERR",
           "faulting_address": 0,
+          "pid": 0,
```

Comment:
Only collect the fields that are relevent for the given signal https://man7.org/linux/man-pages/man2/sigaction.2.html